config/src/test/kotlin/org/springframework/security/config/annotation/web/AuthorizeHttpRequestsDslTests.kt

```diff
@@ -961,4 +964,63 @@ class AuthorizeHttpRequestsDslTests {
         }
 
     }
+
+    @Test
+    fun `request when fully authenticated configured then responds ok`() {
+        this.spring.register(FullyAuthenticatedConfig::class.java).autowire()
+
+        this.mockMvc.post("/path") {
```

Comment:
It doesn't seem necessary that this is a POST. Can you change both of these to a GET? This will also remove the need to add a CSRF token, simplifying the test.

Reply:
Fixed.